internal/index/indexoperations/index.go

```diff
@@ -35,16 +35,16 @@ type Index struct {
 
 // ListIndexes returns a slice of Index objects. The path argument is used as
 // the base path of the index.
-func ListIndexes(path string) ([]Index, error) {
-	dirs, err := ioutil.ReadDir(path)
+func ListIndexes(paths environment.Paths) ([]Index, error) {
+	dirs, err := ioutil.ReadDir(paths.IndexBase())
 	if err != nil {
-		return nil, errors.Wrapf(err, "failed to read directory %s", path)
+		return nil, errors.New("failed to list directory")
```

Comment:
I would expect the path to be part of the error. AFAICT it no longer is.

Reply:
Done.